people-details-sql-jdbc/src/main/java/dao/PersonDAOInterface.java

```diff
@@ -0,0 +1,13 @@
+package dao;
+
+import java.sql.SQLException;
+
+public interface PersonDAOInterface {
+    void create();
+
+    void read();
+
+    void update() throws SQLException;
+
+    void delete();
+}
```

Comment:
Couple of notes here:

1. I think we could have a generic DAO interface and this one will extend it and specify the generic parameter
2. Please decouple the communication with the DB from getting the actual query params (see my explanation in the implementation). I think we should have the following methods (not saying the input params, so you can think about them a bit):

- findAll (however, having such a method cam be quite problematic, pls try to think and explain why)
- findById
- create
- update
- deleteById